lfs/client.go

```diff
@@ -475,7 +508,12 @@ func doHttpRequest(req *http.Request, creds Creds) (*http.Response, error) {
 	}
 
 	if err != nil {
-		err = Error(err)
+		if IsAuthError(err) && isNtlmRequest(res) {
```

Comment:
When would this case come up? It seems like a faulty ntlm server could cause an infinite loop here. Is there some way that `DoNTLMRequest()` could handle this?